docs/dev/development.adoc

```diff
@@ -9,10 +9,16 @@ toc::[]
 
 Requires *Go 1.12*
 
-**WARNING**: If you are adding any features that require a higher version of golang, such as golang 1.13
-for example, please contact maintainers to check of the releasing systems can handle the newer versions.
+**WARNING**: If you are adding any features that require a higher version of golang, that the one mentioned above, please contact maintainers to check of the releasing systems can handle the newer versions.
```

Comment:
Can you put the stuff that user needs to be warned about in an "Admonition" block? Please refer https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#admon-bl.

Reply:
Done, thanks

Reply:
Interesting looks like this does not render in normal view on github but raw view shows it. Il add back the `warning` as well to be sure

Reply:
It's still not rendering well when I look at the rich diff. The list of links is outside the warning box. Here's one way you could do it:
```
[WARNING]
====
If you are adding any features that require a higher version of golang, that the one mentioned above, please contact maintainers to check of the releasing systems can handle the newer versions. If that is ok, please ensure you update the required golang version, both here and in the files below, in your PR.

.List of files to update for golang version
 * link:/scripts/rpm-prepare.sh[`scripts/rpm-prepare.sh`]
 * link:/.travis.yml[`.travis.yml`]
 * link:/Dockerfile.rhel[`Dockerfile.rhel`]
 * link:/openshift-ci/build-root/Dockerfile[`openshift-ci/build-root/Dockerfile`]
====
```